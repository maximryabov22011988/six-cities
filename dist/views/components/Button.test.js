'use strict';
Object.defineProperty(exports, '__esModule', { value: true });
const React = require('react');
const renderer = require('react-test-renderer');
const Button_1 = require('./Button');
it('Button renders correctly with mandatory props', () => {
  const tree = renderer
    .create(React.createElement(Button_1.default, { className: 'login__submit form__submit' }, 'Button'))
    .toJSON();
  expect(tree).toMatchSnapshot();
});
//# sourceMappingURL=Button.test.js.map
