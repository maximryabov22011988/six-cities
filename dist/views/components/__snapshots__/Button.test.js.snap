// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly with mandatory props 1`] = `
<button
  className="login__submit form__submit button"
  type="button"
>
  Button
</button>
`;
